style.css

```diff
@@ -762,7 +766,7 @@
 
 @media screen and (min-width: 768px) {
   .body {
-    width: 90rem;
+    width: 100%;
     height: 320.3rem;
     background: #f4f5f7;
```

Comment:
- [ ] While you did a good job styling the desktop version :+1:, as mentioned before you need to fix some parts, kindly make sure you replicated the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) it means that you should make your Portfolio responsive. Below you can see that there are empty margin when the screen size grows, kindly you should fix it :+1: 

**Your project**

![Screenshot from 2023-02-21 04-02-29](https://user-images.githubusercontent.com/105117832/220299177-3b297dd4-09dc-4449-ad99-427ca7e2dc88.png)

![Screenshot from 2023-02-21 04-02-45](https://user-images.githubusercontent.com/105117832/220299211-57a387f0-b6fc-4d9f-bf11-d1cf646106bc.png)